hardware/raspberrypi/bootmodes/host.md

```diff
@@ -1,6 +1,6 @@
 # USB host boot mode
 
-**USB host boot is available on the Raspberry Pi 3B, 3B+, 3A+ and Raspberry Pi 2B v1.2 models only.**
+**USB host boot is available on the Raspberry Pi 3B, 3B+ and Raspberry Pi 2B v1.2 models only.**
```

Comment:
I've not tested it myself, but I believe the 3A+ should be able to do host-MSD boot, it just can't do host-net boot.

Reply:
Oops - I've edited the wrong file

Reply:
fixed